composer/trainer/dist_strategy.py

```diff
@@ -424,6 +425,7 @@ def _check_fn(module: torch.nn.Module) -> bool:
         print(f'FSDP: Using forward_prefetch={forward_prefetch}')
         print(f'FSDP: Using limit_all_gathers={limit_all_gathers}')
         print(f'FSDP: Using state_dict_type={state_dict_type}')
+        print(f'FSDP: Using sharded_ckpt_prefix_dir={sharded_ckpt_prefix_dir}')
```

Comment:
Can we change all these to `log.info`?

Reply:
ooooh unfortunately that is outside the scope of this PR